zerver/openapi/zulip.yaml

```diff
@@ -22221,6 +22421,34 @@ components:
         - to
         - topic
         - content
+    SavedReply:
+      type: object
+      description: |
+        Object containing the details of the saved reply.
+      additionalProperties: false
+      properties:
+        id:
+          type: integer
+          description: |
+            The unique ID of the saved reply.
+        title:
+          type: string
+          description: |
+            The title of the saved reply.
+        content:
+          type: string
+          description: |
+            The content of the saved reply in text/markdown format.
```

Comment:
I'd continue this with a new paragraph of the form "Clients should insert this content into a message when using a saved reply".